src/styles/annotator/annotator.scss

```diff
@@ -56,6 +56,10 @@ var.$base-font-size: 14px;
     position: relative;
   }
 
+  .is-closed {
+    visibility: hidden;
```

Comment:
There is an issue at the moment that the visual transition to hide the sidebar is animated, but the change in iframe visibility happens immediately. This creates a flicker when closing the sidebar where clicking the "Close" button removes all the sidebar content immediately but the container gradually slides out of view.

The default sidebar open/close transition is only 0.15s so it may be hard to see what is going on, if you slow the transition down, the effect is more obvious.

Reply:
After some digging I learned that you can specify a transition on `visibility`, but since it is a boolean it doesn't animate gradually between the two states. Instead it just [defers the property change](https://stackoverflow.com/questions/27900053/css-transition-with-visibility-not-working) until the end of the transition, which is what we want here.

Interestingly, with this diff:

```diff
diff --git a/src/styles/annotator/annotator.scss b/src/styles/annotator/annotator.scss
index 4cca9199a..a9ddb01f8 100644
--- a/src/styles/annotator/annotator.scss
+++ b/src/styles/annotator/annotator.scss
@@ -11,6 +11,8 @@
 
 var.$base-font-size: 14px;
 
+$sidebar-collapse-transition-time: 2s;
+
 // Sidebar
 .annotator-frame {
   // a CSS reset which attempts to prevent the host page's styles
@@ -54,10 +56,12 @@ var.$base-font-size: 14px;
     width: 100%;
     z-index: 3;
     position: relative;
-  }
 
-  .is-closed {
-    visibility: hidden;
+    transition: visibility $sidebar-collapse-transition-time;
+
+    &.is-closed {
+      visibility: hidden;
+    }
   }
 
   @include icons.icons {
@@ -193,7 +197,7 @@ var.$base-font-size: 14px;
 
 @media screen and (min-width: 37.5em) {
   .annotator-frame {
-    transition: margin-left 0.15s cubic-bezier(0.55, 0, 0.2, 0.8);
+    transition: margin-left $sidebar-collapse-transition-time cubic-bezier(0.55, 0, 0.2, 0.8);
     width: 428px;
     margin-left: -428px;
   }
```

When closing the sidebar, the iframe is not hidden until the end of the transition, which is what we want. When opening the sidebar it is shown immediately however, which is also what we want, but not what I thought would happen (I thought it would wait until the end of the transition before un-hiding the iframe).

Reply:
I'm surprised I missed this, I think I got caught up in tests, but good catch never the less.  

I see, so you're only applying the transition when its collapsing and that will delay the effect until it reaches the end of the delay. Then when it opens, no delay, no transition.